fix(respect): record request bodies in the HAR postData entry (#2992)
Redocly/redocly-cli: .changeset/respect-har-post-data.md

---
'@redocly/cli': patch
---

> Fixed `respect --har-output` recording an empty `postData` for every request. Request bodies are now written to the HAR, so a capture replayed through `drift` can have its request bodies validated instead of silently passing.

Reviewer: ```suggestion
Fixed an issue where `respect --har-output` recorded an empty `postData` for every request.
Request bodies are written to the HAR.
Captures replayed through `drift` can have their request bodies validated instead of silently passing.
```
